r/.Rprofile

```diff
@@ -1,3 +1,3 @@
 source("renv/activate.R")
 for (f in list.files("R", ".R$", full.names = TRUE)) source(f)
-require(dplyr)
+require(dplyr, quietly = T)
```

Comment:
This is weird... the R worker is structured as a package, shouldn't it be used as such? and the require(dplyr) does not seem necessary. But let's at least mute it.

Reply:
if `dplyr` is not necessary, can't we just remove it?